app/views/templates/_form.html.erb

```diff
@@ -84,6 +83,70 @@
       <% end %>
     </div>
 
+    <div class="tab-pane" id="help">
+      <div>
+        <%= _('Provisioning templates use ERB syntax') %>
```

Comment:
Sure, I will drop `With safemode disabled other macros may work, but are not supported at this time.` as it is not that relevant and only show the last sentence when appropriate. FYI the setting is named "Safemode rendering" but the description says "safe mode" - I will follow that. Pushing in a minute.